src/pages/index.tsx

```diff
@@ -1,83 +1,47 @@
+import Card from "@pb/components/card";
+import HeadMeta from "@pb/components/head-meta";
+import Header from "@pb/components/header";
 import { type NextPage } from "next";
-import Head from "next/head";
-import Link from "next/link";
-import { signIn, signOut, useSession } from "next-auth/react";
+import { type CardData } from "@pb/types";
```

Comment:
Could you kindly ensure that there is a space between the various import types, please?